main.go

```diff
@@ -21,8 +21,9 @@ import (
 func main() {
 	inPath := flag.String("in", "-", "input file, url, or - for stdin")
 	outPath := flag.String("out", "-", "output file, or - for stdout")
+	method := flag.String("method", "nearest", "image resize method - 'nearest', 'bicubic', 'bilinear', 'mitchell', 'lanczos2', or 'lanzoc3'")
```

Comment:
When i tested this, it just got ignored and used default, but it's probably best to throw something.